book-crawler/scripts/monitor_db.py

```
@@ -0,0 +1,261 @@
+            duplicates = self.db.query(
+                Book.isbn_13,
+                func.count(Book.id).label('count')
+            ).group_by(Book.isbn_13).having(func.count(Book.id) > 1).all()
+            
+            if duplicates:
+                for isbn, count in duplicates:
+                    issues.append(f"ISBN 중복: {isbn} ({count}개)")
+            
+            # 2. 필수 필드 누락 체크
+            missing_title = self.db.query(func.count(Book.id)).filter(
+                (Book.book_name == None) | (Book.book_name == '')
```

Ruff (0.12.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)